tests/testthat/test-derive_param_extreme_event.R

```diff
@@ -0,0 +1,178 @@
+adsl <- tibble::tribble(
+  ~USUBJID, ~DTHDT,
+  "1",      ymd("2022-05-13"),
+  "2",      ymd(""),
+  "3",      ymd("")
+) %>%
+  mutate(STUDYID = "XX1234")
+
+adrs <- tibble::tribble(
+  ~USUBJID, ~ADTC,        ~AVALC, ~PARAMCD,
+  "1",      "2020-01-02", "PR",   "OVR",
+  "1",      "2020-02-01", "CR",   "OVR",
+  "1",      "2020-03-01", "CR",   "OVR",
+  "1",      "2020-04-01", "SD",   "OVR",
+  "2",      "2021-06-15", "SD",   "OVR",
+  "2",      "2021-07-16", "PD",   "OVR",
+  "2",      "2021-09-14", "PD",   "OVR",
+  "1",      "2020-01-02", "PR",   "OVRF",
+  "1",      "2020-02-01", "CR",   "OVRF",
+  "1",      "2020-03-01", "CR",   "OVRF",
+  "1",      "2020-04-01", "PD",   "OVRF",
+  "2",      "2021-06-15", "SD",   "OVRF",
+  "2",      "2021-07-16", "PD",   "OVRF",
+  "2",      "2021-09-14", "PD",   "OVRF"
+) %>%
+  mutate(
+    STUDYID = "XX1234",
+    ADT = ymd(ADTC)
+  ) %>%
+  select(-ADTC)
+
+# derive_param_extreme_event ----
+## derive_param_extreme_event Test 1: derive first PD date ----
```

Comment:
The comment is not following our convention. Calling the ["Format test_that test file" addin](https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#addin-admiraldevformat_test_that_file) should fix it.